src/EventStore.Common.Tests/Configuration/ConfigurationRootExtensionsTest.cs

```diff
@@ -47,4 +52,47 @@ public class ConfigurationRootExtensionsTest {
 			EventStore.Common.Configuration.ConfigurationRootExtensions.GetCommaSeparatedValueAsArray(
 				configuration, "GossipSeed"));
 	}
+
+	[Fact]
+	public void user_specified_config_file_through_environment_variables_returns_true() {
+		
+		IDictionary environmentVariables = new Dictionary<string, string>();
+		environmentVariables.Add("EVENTSTORE_CONFIG", "pathToConfigFileOnMachine");
+
+		_configurationRoot = new ConfigurationBuilder()
+			.Add(new EnvironmentVariablesSource(environmentVariables))
+			.Build();
+
+		var result = ConfigurationRootExtensions.IsUserSpecified(_configurationRoot, _configFileKey);
+		
+		Assert.True(result);
+	}
+	
+	[Fact]
+	public void user_specified_config_file_through_command_line_returns_true() {
+
+		var args = new string[] {
+			"--config=pathToConfigFileOnMachine"
+		};
+
+		_configurationRoot = new ConfigurationBuilder()
+			.Add(new CommandLineSource(args))
+			.Build();
+
+		var result = ConfigurationRootExtensions.IsUserSpecified(_configurationRoot, _configFileKey);
```

Comment:
```suggestion
		var result = _configurationRoot.ConfigurationRootExtensions.IsUserSpecified(_configFileKey);
```

just cause this is the way we're expecting the method to really be used (same in the other tests too)